dashboard-ui/src/components/App/index.jsx

```diff
@@ -296,7 +296,13 @@ export class App extends React.Component {
                     });
                 }
                 else {
-                    const newPid = Math.max(...pLog.filter((x) => x.Type === classification && !["202409113A", "202409113B"].includes(x['ParticipantID'])).map((x) => Number(x['ParticipantID']))) + 1;
+                    // still want to record distinction between civ and mil but it should no longer effect the actual pid
+                    const newPid = Math.max(...pLog.filter((x) => 
+                        (classification === 'Mil' || classification === 'Civ' ? 
+                            (x.Type === 'Mil' || x.Type === 'Civ') : 
+                            x.Type === classification) && 
```

Comment:
I'm not sure any of this classification stuff is needed. I'd remove it for now and then later when we add the online classification we can update this, but I think we just want to filter out those two strings and that's it

Reply:
True. I'll leave in the filter for ["202409113A", "202409113B"] because Mongo says those are the max numbers but remove the rest of the filter